packages/block-library/src/blocks/posts-collection/style.scss

```diff
@@ -0,0 +1,108 @@
+.novablocks-collection__layout[data-layoutstyle="carousel"] {
+  --novablocks-collection-layout-spacing: 0;
+}
+
+[data-carousellayout="variable"] {
+  
+  .novablocks-card__layout {
+    display: table;
+    width: 1px;
+  }
+  
+  .novablocks-card--fixed-media-aspect-ratio .novablocks-card__media {
```

Comment:
Use `.novablocks-collection-carousel--variable` class instead of `[data-carousellayout="variable"]` in order to not have duplicate styles in editor-styles
https://github.com/pixelgrade/nova-blocks/pull/290/files#diff-335eda720cfc2ba6043dd7e45dce771d5c718d082584613623904815d5a7b378R1